autowrap/ConversionProvider.py

```diff
@@ -276,8 +300,41 @@ def output_conversion(self, cpp_type, input_cpp_var, output_py_var):
         return "%s = <%s>%s" % (output_py_var, cpp_type, input_cpp_var)
 
 
+class UnsignedIntegerConverter(TypeConverterBase):
+
+    """
+    wraps long and int. "long" base_type is converted to "int" by the
```

Comment:
is this relevant to the long unique ids we use?

Reply:
Don't know. But this was like this before. I just added a new converter that also checks positivity.